src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetCoreIntegration.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Reflection;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.Integrations
+{
+    /// <summary>
+    /// Tracer integration base for core web server integrations.
+    /// </summary>
+    public static class AspNetCoreIntegration
+    {
+        private const string IntegrationName = "AspNetCore";
+        private static readonly ILog Log = LogProvider.GetLogger(typeof(AspNetCoreIntegration));
+
+        /// <summary>
+        /// Entry method for invoking the beginning of every web server request pipeline
+        /// </summary>
+        /// <param name="httpContext">Instance being instrumented.</param>
+        /// <param name="features">Initialize features.</param>
+        /// <param name="opCode">The OpCode used in the original method call.</param>
+        /// <param name="mdToken">The mdToken of the original method call.</param>
+        [InterceptMethod(
+            TargetAssembly = "Microsoft.AspNetCore.Http.Abstractions",
+            TargetType = "Microsoft.AspNetCore.Http.DefaultHttpContext",
+            TargetSignatureTypes = new[] { ClrNames.Void, ClrNames.Ignore })]
```

Comment:
I elected to ignore the type of the parameter because we don't rely on it whatsoever.
That will keep this a bit more resilient to any renames, as a rename would not affect us.
Thoughts on that?

Reply:
Also, should I add an explicit version range here?

Reply:
I personally don't think the version range / strong typing is necessary if we're getting the correct metadata token. Should there be a new overload of Initialize it will be called with the argument of the new type.

Reply:
I think we still need the version range and signature checks. In the profiler code, we need to know that the method we're about to replace has the expected arguments (i.e. matches the wrapper), otherwise we will cause an exception. Having the original metadata token doesn't change this.